src/channeldef.ts

```diff
@@ -893,7 +893,8 @@ export function verbalTitleFormatter(fieldDef: FieldDefBase<string>, config: Con
     } else if (isArgminDef(aggregate)) {
       return `${field} for min ${aggregate.argmin}`;
     } else {
-      return `${titleCase(aggregate)} of ${field}`;
+      const aggregateOp = isExponentialDef(aggregate) ? 'exponential' : aggregate;
```

Comment:
I ended up adding a mixture of ternary operators and if statements into the code to handle `exponential` being an object which I know can increase code complexity. Please let me know if there are more clean ways to handle this :)